mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -169,11 +177,20 @@
         <CmdButtonArray index="Execute">
             <Flags index="ShowInGlossary" value="0"/>
         </CmdButtonArray>
+        <InfoArray index="0">
+            <SectionArray index="Stats">
+                <DurationArray index="Duration" value="0"/>
+            </SectionArray>
+        </InfoArray>
+    </CAbilMorph>
+    <CAbilMorph id="BurrowLurkerMPDown">
+        <InfoArray index="0" RandomDelayMax="0.125"/>
     </CAbilMorph>
     <CAbilMorph id="BurrowLurkerMPUp">
         <CmdButtonArray index="Execute">
             <Flags index="ShowInGlossary" value="0"/>
         </CmdButtonArray>
+        <InfoArray index="0" RandomDelayMax="0"/>
```

Comment:
undocumented: Lurker's Unburrow random starting delay was changed from [0, 0.5] to [0, 0]
This means Lurkers will be able to unburrow and move away up to one second faster